src/GeometryOps.jl

```diff
@@ -2,11 +2,19 @@
 
 module GeometryOps
 
-include("../GeometryOpsCore/src/GeometryOpsCore.jl") # TODO: replace this with `using GeometryOpsCore`
-import .GeometryOpsCore
-for name in setdiff(names(GeometryOpsCore, all = true), (:eval, :var"#eval", :include, :var"#include"))
+import GeometryOpsCore
+for name in Base.setdiff(
```

Comment:
This is pretty obscure, is there a less scary way to do this so regular contributors don't run away straight away?

Personally I would just make a single tuple with everything you want to import and skip the union/setdiff part.

We should also export everything explicitly so people can read it

Reply:
Ah yeah that's a good point.  I can probably just explicitly import stuff.